src/main/java/org/assertj/core/api/AbstractCharSequenceAssert.java

```diff
@@ -361,6 +361,80 @@ public SELF hasSize(int expected) {
     return myself;
   }
 
+  /**
+   * Verifies that the actual {@code CharSequence} has a maximum length using the {@code length()}  method.
+   * <p>
+   * This assertion will succeed:
+   * <pre><code class='java'>assertThat(&quot;A Game of Thrones&quot;).hasSizeLessThan(18);</code></pre>
+   *
+   * Whereas this assertion will fail:
+   * <pre><code class='java'>assertThat(&quot;A Game of Thrones&quot;).hasSizeLessThan(17);</code></pre>
+   *
+   * @param expected the expected maximum length of the actual {@code CharSequence}.
+   * @return {@code this} assertion object.
+   * @throws AssertionError if the actual length is equal or greater than the expected length.
+   */
+  public SELF hasSizeLessThan(int expected) {
+    strings.assertHasSizeLessThan(info, actual, expected);
+    return myself;
+  }
+
+  /**
+   * Verifies that the actual {@code CharSequence} has a length less than or equal to the given value using the
+   * {@code length()}  method.
+   * <p>
+   * This assertion will succeed:
+   * <pre><code class='java'>assertThat(&quot;A Game of Thrones&quot;).hasSizeLessThanOrEqualTo(17);</code></pre>
+   *
+   * Whereas this assertion will fail:
+   * <pre><code class='java'>assertThat(&quot;A Game of Thrones&quot;).hasSizeLessThanOrEqualTo(18);</code></pre>
```

Comment:
if `hasSizeLessThanOrEqualTo(17)` works then `hasSizeLessThanOrEqualTo(18)` should to ;-)